src/Microsoft.OData.Core/Batch/ODataBatchUtils.cs

```diff
@@ -13,7 +13,6 @@ namespace Microsoft.OData
     using System.Globalization;
     using System.IO;
     using System.Linq;
-    using Microsoft.OData.JsonLight;

```

Comment:
Was this namespace renamed to something else or is it just not being used in this file?

Reply:
It was renamed to `using Microsoft.OData.Json`